website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "NodeClaims"
+linkTitle: "NodeClaims"
+weight: 30
+description: >
+  Understand NodeClaims
+---
+
+Karpenter uses NodeClaims to perform node autoscaling based on the requests of unschedulable pods and the
+requirements of existing [NodePool]({{<ref "./nodepools" >}}) and associated [NodeClasses]({{<ref "./nodeclasses" >}}).
+While NodeClaims require no direct user input, as a Karpenter user you can monitor NodeClaims to keep track of
+the status of your nodes in cases where something goes wrong or a node drifts from its intended state.
+
+Karpenter uses NodeClaims as a merged representation of the cloud provider instance and the node on the cluster.
```

Comment:
I see it as "NodeClaim is the request for capacity" and "Node is the running representation of the capacity". I don't see them as merged. NodeClaims cause Nodes as a side effect of launching the instance.

Reply:
Done.